pypesto/objective/amici_objective.py

```diff
@@ -362,8 +367,21 @@ def _call_amici(
                 res = np.hstack([res, rdata['res']]) \
                     if res.size else rdata['res']
                 if sensi_order > 0:
-                    sres = np.vstack([sres, rdata['sres']]) \
-                        if sres.size else rdata['sres']
+                    opt_sres = sim_sres_to_opt_sres(
+                        self.x_ids,
+                        self.mapping_par_opt_to_par_sim[data_ix],
+                        rdata['sres'],
+                        coefficient=1.0
```

Comment:
We have now always written it s.t. here ``-1`` has to be passed.

Reply:
I don't get this comment

Reply:
e.g. in line 355, there is a coefficient of ``-1`` inserted, because we pass the gradient of the llh, but compute the nllh, so everything has to be minused

Reply:
yes, but for the residuals thats not true, we always return `res` not `nres`

Reply:
right. my mistake :)